streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsKafkaClientTest.java

```diff
@@ -124,6 +131,78 @@ public void shouldNotAllowNullTopicConfigs() throws Exception {
         verifyCorrectTopicConfigs(streamsKafkaClient, topicConfigWithNoOverrides, Collections.singletonMap("cleanup.policy", "delete"));
     }
 
+    @Test
+    public void testCheckBrokerCompatibilityEmptyResponse() {
+        kafkaClient.prepareResponse(null);
+        final StreamsKafkaClient streamsKafkaClient = createStreamsKafkaClient();
+        try {
+            streamsKafkaClient.checkBrokerCompatibility(false);
+            fail("Response was not empty");
+        } catch (final StreamsException se) {
+            assertThat(se.getMessage(), equalTo("Empty response for client request."));
```

Comment:
nit: I don't think we should bother checking the string messages as it is quite brittle. The important thing is that the exception is thrown